ember.md

```diff
@@ -98,6 +100,40 @@ get(this, 'isDestructured'); //true
 set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
+
+<a name="data"></a>
+## Data
+
+<a name="data--data-binding"></a>
+### 3.1 Data Binding
+
+Any new code written by ASH should only contain one-way data-binding. Third-party addons using two-way data-binding is ok, but be cautious and conscious of the side effects it can have.
```

Comment:
@ASH-Anthony mentioned the need to "guard against" two-way binding in third party addons. I don't think this is touched in here.